data/remnant/remnant ships.txt

```diff
@@ -39,6 +39,8 @@ ship "Albatross"
 		"outfit scan power" 18
 		"outfit scan speed" 1
 		"tactical scan power" 39
+		"hull protection" 1
+		"shield protection" 1
```

Comment:
Unintentional? I didn't see this mentioned in the summary.

Reply:
Yeah, this is unintentional. Not sure why it's here at all, but I suspect it was an accidental thing, testing to see if they are displayed.

Thanks for commenting on it!

Reply:
Removed. Thanks for pointing it out!